config/sys.config.src

```diff
@@ -12,7 +12,9 @@
             {app_id, "${PP_CHIRPSTACK_APP_ID}"},
             {api_key, "${PP_CHIRPSTACK_API_KEY}"},
             {app_eui, "${PP_CHIRPSTACK_APP_EUI}"}
-        ]}
+        ]},
+        {accept_joins, "${PP_ACCEPT_JOINS}"},
+        {net_ids, ${PP_NET_IDS}}
```

Comment:
Can I substitute without wrapping it in a string?
I was trying to avoid having to parse integers in a list every time a packet comes through.